assets/dso-toolkit-scripts.js

```diff
@@ -7,3 +11,136 @@ function DsoToolkitResizeIframe(frame, element) {
 
   frame.style.height = (height + 32) + 'px';
 }
+
+(function () {
+  var cleanRegex = /[ -]/g;
+
+  Vue.component('dso-navigation-filter', {
+    template:
+      '<div class="dso-search-bar">' +
+        '<div class="dso-search-bar-input">' +
+          '<span class="dso-search-icon" aria-hidden="true"></span>' +
+          '<input type="text" placeholder="Filter op component" v-model.trim="input">' +
+          '<button type="button" @click="clearFilter()" title="Zoekopdracht legen">' +
+            'Zoekopdracht legen' +
+          '</button>' +
+        '</div>' +
+      '</div>',
+    data: function () {
+      return {
+        input: localStorage.getItem('filterValue') || '',
+        items: Array.prototype.slice.call(document.querySelectorAll('[data-role="item"]')),
+        collections: Array.prototype.slice.call(document.querySelectorAll('[data-behaviour="collection"]'))
```

Comment:
Werkt dit wel? Nog voordat ik naar je code had gekeken heb ik in het issue al opgemerkt:

> Verder, het komt nu vrij snel voor dat er met een zoekterm een hele categorie (dus: FUNDAMENT, COMPONENTEN, of VOORBEELDEN) compleet leeg blijft. Misschien zouden we dan die hele kop niet willen zien?

Blijkbaar is dat ook precies wat je wil, maar, werkt het (nog?) niet?

Reply:
Laat maar: @bramhoosemans geeft aan dat hij het prima vindt dat deze koppen blijven staan.